libs/dh/esett/shell/src/lib/outgoing-messages/status-badge/dh-outgoing-message-status-badge.component.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { TranslocoModule } from '@ngneat/transloco';
+
+import { WattBadgeComponent } from '@energinet-datahub/watt/badge';
+import { DocumentStatus } from '@energinet-datahub/dh/shared/domain/graphql';
+import { DhEmDashFallbackPipe } from '@energinet-datahub/dh/shared/ui-util';
+
+@Component({
+  standalone: true,
+  selector: 'dh-outgoing-message-status-badge',
+  template: `
+    <ng-container [ngSwitch]="status" *transloco="let t; read: 'eSett.status'">
```

Comment:
nitpick: let's reuse the translations under 'outgoingMessages.shared.documentStatus'